admin_guide/install/docker_registry.adoc

```diff
@@ -106,20 +90,22 @@ containers, each with its own local storage. This will lead to unpredictable
 behavior, as subsequent requests to pull the same image repeatedly may succeed
 some times and fail others, depending on which container the request ultimately
 goes to.
+====
 
-- The `--mount-host` options requires that the registry container run in
-privileged mode. This is automatically enabled for you when you specify
-`--mount-host`. However, by default, not all pods are allowed to run privileged
-containers. If you still want to use this option:
-+
-. Create a service account for the registry to run as:
+The `--mount-host` option requires that the registry container run in privileged
+mode. This is automatically enabled for you when you specify `--mount-host`.
+However, not all pods are allowed to run privileged containers by default. If
+you still want to use this option:
+
+. Create a new link:../service_accounts.html[service account] in the *default*
+project for the registry to run as. The following example creates a service
+account named *registry*, but it can be named anything:
 +
 ----
 $ echo \
-    '{"kind":"ServiceAccount","apiVersion":"v1","metadata":{"name":"registry"}}' \//<1>
+    '{"kind":"ServiceAccount","apiVersion":"v1","metadata":{"name":"registry"}}' \
     | oc create -f -
```

Comment:
Should we make this "oc create -n default -f -" per the previous comment, or is the above context enough?